Sources/SwiftDocC/Utility/MarkupExtensions/BlockDirectiveExtensions.swift

```diff
@@ -17,4 +17,22 @@ extension BlockDirective {
         Tutorial.directiveName,
         TutorialArticle.directiveName,
     ]
+
+    /// Names of known directives
+    static let allKnownDirectiveNames: [String] = [
```

Comment:
I think we're missing quite a few directives here. If I search for "public static let directiveName" in the repo, I'm finding 30 `DirectiveConvertible` structs.

Reply:
Good one Ethan, i updated it to capture them all now.